dbr_test.go

```diff
@@ -100,73 +131,78 @@ func reset(t *testing.T, sess *Session) {
 
 func TestBasicCRUD(t *testing.T) {
 	for _, sess := range testSession {
-		reset(t, sess)
```

Comment:
There's no big change here, I just changed the test name to include the dialect, otherwise it's difficult to see which one was failing.

Reply:
Changed all tests to be this way in https://github.com/gocraft/dbr/pull/268/files